packages/eslint-plugin-pf-codemods/src/rules/v6/tileDeprecated/tile-deprecated.test.ts

```diff
@@ -0,0 +1,22 @@
+const ruleTester = require('../../ruletester');
+import * as rule from './tile-deprecated';
+
+ruleTester.run('tile-deprecated', rule, {
+  valid: [
+    {
+      code: `import { Tile } from '@someOtherPackage';`,
+    },
+  ],
+  invalid: [
```

Comment:
Can we just add the following tests:

- Tile import aliased
- Tile imports from dist (esm, js, and dynamic paths; the bannerReplaceVariantProp rule should have tests for reference)